airbyte-webapp/.stylelintrc

```diff
@@ -12,7 +12,8 @@
     "scss/dollar-variable-empty-line-before": null,
     "scss/dollar-variable-pattern": null,
     "scss/percent-placeholder-pattern": null,
-    "value-keyword-case": null
+    "value-keyword-case": null,
+    "color-no-hex": true
   },
-  "ignoreFiles": ["**/build/**", "**/dist/**"]
+  "ignoreFiles": ["**/build/**", "**/dist/**", "**/**/_colors.scss", "**/**/index.css"]
```

Comment:
Could we not add those files completely from stylelinting, but please instead use [disable comments](https://stylelint.io/user-guide/ignore-code/#within-files) (similar to how we also handle it with eslint), in those two files on the top, to just disable this specific rule from running in those files. So we can also leave a comment in those files, why colors are allowed in this specific files.

Reply:
Oh, okay, make sense 👍🏻 will fix that